examples/ensemble/plot_gradient_boosting_regression.py

```diff
@@ -19,38 +25,81 @@
 
 from sklearn import ensemble
 from sklearn import datasets
-from sklearn.utils import shuffle
 from sklearn.metrics import mean_squared_error
+from sklearn.model_selection import train_test_split
+
+##############################################################################
+# Load the data
+# -------------------------------------
+#
+# First we need to load the data. We will work on both datasets
+# simultaneously. We set random state to be consistent with the result.
+
+cancer = datasets.load_breast_cancer()
+X, y = cancer.data, cancer.target
+
+##############################################################################
+# Data preprocessing
+# -------------------------------------
+#
+# Next, we will split our dataset to 90% for training and leave the rest for
+# testing. We will also prepare the parameters we want to use to fit our
+# regression model. You can play with those parameters to see how the
+# results change:
+#
+# Here:
+# n_estimators : is the number of boosting stages which will be performed.
+#     Later, we will plot and see how the deviance changes with those boosting
+#     operations.
+# max_depth : this limits the number of nodes in the tree. The best value
+#     depends on the interaction of the input variables.
+# min_samples_split : is the minimum number of samples required to split an
+#     internal node.
+# learning_rate: tells how much the contribution of each tree will shrink
+# loss: here, we decided to use least squeares as a loss function, however
+#     there are many other options (check
+#     :class:`~sklearn.ensemble.GradientBoostingRegressor` to see what are
+#     other possibilities)
+# In this example we will use the same parameters for both datasets even if
+# they might not be the most optimal
```

Comment:
Here again I do not understand what "both datasets" refer to. Maybe just drop this last sentence.